### ISSUE TITLE
1-01-01 Observed variables (atmosphere), corrections

### ISSUE THREAD

**Issue by fstuerzl** [wmo-im/wmds#253]
**Summary and Purpose**
Variables in \Atmosphere\Gas Branch need to be corrected.
 
**Stakeholder(s)**
@joergklausen 

**Proposal**
Correct names and paths and add descriptions as follows (changes indicated in bold):
notation | path | name | description
--|--|--|--
283 | \Atmosphere\Gas\Reactive Gas\CHCl3 (**trichloromethane,** chloroform) | CHCl3 (**trichloromethane,** chloroform) | **IUPAC:   chloroform, PubChem CID: 6212, CAS Number: 67-66-3**
489 | \Atmosphere\Gas\Reactive Gas\VOC\CH2O (**methanol,** formaldehyde) | CH2O (**methanol,** formaldehyde) | **IUPAC: formaldehyde, PubChem CID: 712, CAS Number: 50-00-0**
536| \Atmosphere\Gas\Greenhouse Gas\**HFCs**\C2H3F3 (1,1,1-trifluoroethane, **HFC-143a)** | C2H3F3 (1,1,1-trifluoroethane, **HFC-143a**) | **IUPAC: 1,1,1-trifluoroethane, PubChem CID: 9868, CAS Number: 420-46-2**

**Reason**
The terms trichloromethane and methanol were missing in the names for variables 283 and 489. Variable 536 C2H3F3 is a HFC, and not a HCFC. This should be changed in the variable name and path. 

Branch: _(will be added after FT-2021-1)_

**Comment by sebvi** [comment]
@fstuerzl 
just a quick comment: methanol is not correct. The chemical formula of methanol is CH3OH, maybe you intended to write methan**a**l which is the other name of formaldehyde?

**Comment by fstuerzl** [comment]
Merci, @sebvi! That's right of course. I've edited the proposal.

**Comment by gaochen-larc** [comment]
One question: Is "reactive gas" definition based on atmospheric lifetime?   CH2O's lifetime is a few hours and CHCl3's lifetime is a few months.   The both classified as reactive gas.

**Comment by joergklausen** [comment]
The complement is 'greenhouse gas', and 'other gas'. It is the conventional way of grouping gases in GAW. 'Other gas' only lists hydrogen, which is traditionally dealt with by the SAG GHG. This is how this grouping came about ... there may be better ways of organizing this, and good ideas are of course welcome.

**Comment by gaochen-larc** [comment]
> The complement is 'greenhouse gas', and 'other gas'. It is the conventional way of grouping gases in GAW. 'Other gas' only lists hydrogen, which is traditionally dealt with by the SAG GHG. This is how this grouping came about ... there may be better ways of organizing this, and good ideas are of course welcome.

"Other gas" branch will eliminate the lifetime issue.  The only downside is this category will have a very large number of compounds.  We may try to group trace gases by composition, like nitrogen, sulfur.  The carbon compounds need additional branches, like alkane, alkene...  NASA GCMD has a similar scheme.  We followed this scheme: https://www-air.larc.nasa.gov/missions/etc/AtmosphericCompositionVariableStandardNames.pdf .

**Comment by gaochen-larc** [comment]
Variable name, molecular formula, PubChem CID, and CAS number are confirmed.  Thank you Franziska! 

The upper level branch structure needs to be discussed.

**Comment by joergklausen** [comment]
Structure confirmed, branch confirmed, please proceed.
NB: The overall organization of variables in the sub-domain 'gas' is currently under review. At present, the paths are neither included in the codes registry nor in the description, so these changes can be made independently.

**Comment by fstuerzl** [comment]
To move forward with this issue, we need another confirmation. @dkubisti, could you also take a look at the branch, please, and state, whether the changes (aside from the paths) are correct?

**Comment by dkubisti** [comment]
Yes, I confirm variable name, molecular formula, PubChem CID, and CAS number. Thank you.

**Comment by amilan17** [comment]
ready for FT
